encryption/symmetric.go

```diff
@@ -52,12 +52,15 @@ func NewDefaultKDFParams(KDF string) (*KDFParams, error) {
 		return &KDFParams{
 			KDF:  "ARGON2ID",
 			Salt: salt,
-			Argon2IDParams: &Argon2IDParams{
+			Argon2idParams: &Argon2idParams{
 				SaltSize: saltSize,
 				Time:     4,
-				Memory:   256 * 1024,
-				Threads:  uint8(runtime.NumCPU()),
-				KeyLen:   32,
+				// from: https://pkg.go.dev/golang.org/x/crypto/argon2
+				// [...] the memory parameter specifies the size of the memory in KiB.
+				// For example memory=64*1024 sets the memory cost to ~64 MB
+				Memory:  256 * 1024,
+				Threads: uint8(runtime.NumCPU()),
```

Comment:
so by reading the other issue i guess this has to be changed to one to actually reflect CRYPTOGRAPHY.md

Reply:
oops